ambiguous argument / panic

I accidentally tried running "git-pr" inside a cloned "git-pr" repo:

```
~/tmp/git-pr$ git-pr
fatal: ambiguous argument 'origin/main..head': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

panic: failed to find common ancestor for origin/main and head: exit status 128

goroutine 1 [running]:
main.must[...](...)
	/home/duane/tmp/git-pr/utils.go:49
main.main()
	/home/duane/tmp/git-pr/main.go:36 +0x125c
```

I'm new to git-pr so I'm not sure what's going on, but I thought it might be something that needs a boost so it can be more informative if this happens to others in a more serious situation.

--- a/main.go
+++ b/main.go
@@ -14,7 +14,7 @@ import (
 
 const (
 	KeyRemoteRef = "remote-ref"
-	head         = "head"
+	head         = "HEAD"
 )
 
 var emojis0 = []string{"♈️", "♉️", "♊️", "♋️", "♌️", "♍️", "♎️", "♏️", "♐️", "♑️", "♒️", "♓️"}

